inst/qml/Ancova.qml

```diff
@@ -235,6 +235,23 @@ Form
 		}
 	}
 	
+	Section
+	{
+		title: qsTr("Raincloud Plots")
+
+		VariablesForm
+		{
+			preferredHeight: 150 * preferencesModel.uiScale
+			AvailableVariablesList { name: "rainCloudPlotsVariables";		title: qsTr("Factors"); source: ["fixedFactors", "randomFactors"] }
+			AssignedVariablesList { name: "rainCloudPlotsHorizontalAxis";	title: qsTr("Horizontal Axis"); singleVariable: true }
+			AssignedVariablesList { name: "rainCloudPlotsSeparatePlots";	title: qsTr("Separate Plots");	singleVariable: true }
+		}
+		CheckBox
+		{
+			name: "rainCloudPlotsHorizontalDisplay"; label: qsTr("Horizontal display")
+		}
```

Comment:
So this should really be a common QML element!

Take a look at https://github.com/jasp-stats/jaspDistributions/tree/master/inst/qml, for an example of how to do this.

There are QML files for the analyses, and a folder called `common` for the elements that are reused between analyses.

I suggest that you

1. Make folder called `common`.
2. Make a file for the common qml code, maybe RaindCloudPlots.qml?
3. At the top of each file, write `import "./common" as ANOVA` (or instead of ANOVA, whatever name you prefer).
4. Then you can do `ANOVA.RainCloudPlots` and everything should work.

Note that this general implementation could take some arguments. For example, the source could just be a parameter set in the individual analyses.

Reply:
I made a common element as you described now! Let's see what the unit tests say.